ssl/t1_trce.c

```diff
@@ -1278,7 +1278,8 @@ void SSL_trace(int write_p, int version, int content_type,
             int hvers;
 
             /* avoid overlapping with length at the end of buffer */
-            if (msglen < (SSL_IS_DTLS(ssl) ? 13 : 5)) {
+            if (msglen < (size_t)(SSL_IS_DTLS(ssl) ? 
+                DTLS1_RT_HEADER_LENGTH : SSL3_RT_HEADER_LENGTH)) {
```

Comment:
this line should be indented by 4 spaces
and the following lines should be out-dated (or what is the right english word?)